addons/pvr.vdr.vnsi/src/VNSIData.cpp

```diff
@@ -809,6 +808,164 @@ PVR_ERROR cVNSIData::GetRecordingEdl(const PVR_RECORDING& recinfo, PVR_EDL_ENTRY
   return PVR_ERROR_NO_ERROR;
 }
 
+int cVNSIData::GetDeletedRecordingsCount()
```

Comment:
pvr.wmc needs version bumped in its clientversion.h file as well please
On 07/02/2015 7:52 pm, "Rainer Hochecker" <notifications@github.com> wrote:

> In addons/pvr.vdr.vnsi/src/VNSIData.cpp
> <https://github.com/opdenkamp/xbmc-pvr-addons/pull/427#discussion_r24288489>
> :
>
> > @@ -809,6 +808,164 @@ PVR_ERROR cVNSIData::GetRecordingEdl(const PVR_RECORDING& recinfo, PVR_EDL_ENTRY
> >    return PVR_ERROR_NO_ERROR;
> >  }
> >
> > +int cVNSIData::GetDeletedRecordingsCount()
>
> 1) please put the implementation into a separate commit.
> 2) you need to bump vnsi protocol when adding new functions. then check if
> those functions are available on the backend before using those.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opdenkamp/xbmc-pvr-addons/pull/427/files#r24288489>.
>